aws/network-primitives/scripts/create_bootstrap_node_evm_compose_file.sh

```diff
@@ -119,17 +123,19 @@ cat >> ~/subspace/docker-compose.yml << EOF
       "--prometheus-external",
 EOF
 
-for (( i = 0; i < node_count; i++ )); do
-  if [ "${current_node}" != "${i}" ]; then
-    addr=$(sed -nr "s/NODE_${i}_MULTI_ADDR=//p" ~/subspace/node_keys.txt)
+
+for (( i = 0; i < bootstrap_node_count; i++ )); do
```

Comment:
This would also set self node key as reserved. I dont think that is what we want

Reply:
This is the bootstrap Domain Operator node